cypress/integration/patch/configure-patch.ts

```diff
@@ -178,6 +179,49 @@ describe("Configure Patch Page", () => {
         });
     });
   });
+  describe("Scheduling a patch", () => {
+    beforeEach(() => {
+      cy.login();
+      cy.listenGQL();
+    });
+    it("Clicking `Schedule` button schedules patch and redirects to patch page", () => {
+      cy.visit(`/patch/${unactivatedPatchId}`);
+      const val = "hello world";
+      cy.get(`[data-cy=configurePatch-nameInput]`)
+        .as("patchNameInput")
+        .clear()
+        .type(val);
+      cy.route({
+        method: "POST",
+        url: "/graphql/query",
+        response: mockedSuccessfulConfigureResponse,
+      });
+      cy.get("[data-cy=schedule-patch]").click();
+      cy.location().should((loc) =>
+        expect(loc.pathname).to.eq(`/patch/${unactivatedPatchId}/tasks`)
+      );
+    });
+    it("Shows error banner if unsuccessful and keeps data", () => {
+      cy.visit(`/patch/${unactivatedPatchId}`);
+      const val = "hello world";
+      cy.get(`[data-cy=configurePatch-nameInput]`)
+        .as("patchNameInput")
+        .clear()
+        .type(val);
+      cy.route({
+        method: "POST",
+        url: "/graphql/query",
+        response: mockedErrorConfigureResponse,
+      });
+      cy.get("[data-cy=schedule-patch]").click();
+      cy.location().should((loc) =>
+        expect(loc.pathname).to.include(
```

Comment:
nit. We could prob just compare the path names directly.

Reply:
so true